packages/lens/documents/queries/UnknownOpenActionData.graphql

```diff
@@ -0,0 +1,40 @@
+query UnknownOpenActionData($module: EvmAddress!, $pubId: PublicationId!) {
+  publication(request: { forId: $pubId }) {
+    ... on Post {
+      stats {
+        countOpenActions(request: { anyOf: { address: $module } })
+      }
+      operations {
+        actedOn(request: { filter: { address: $module } }) {
+          ... on UnknownOpenActionResult {
+            address
+          }
+        }
+      }
+    }
+    ... on Comment {
+      stats {
+        countOpenActions(request: { anyOf: { address: $module } })
+      }
+      operations {
+        actedOn(request: { filter: { address: $module } }) {
+          ... on UnknownOpenActionResult {
+            address
+          }
+        }
+      }
+    }
+    ... on Quote {
+      stats {
+        countOpenActions(request: { anyOf: { address: $module } })
+      }
+      operations {
+        actedOn(request: { filter: { address: $module } }) {
+          ... on UnknownOpenActionResult {
+            address
+          }
+        }
+      }
+    }
+  }
+}
```

Comment:
We should not make an extra graphql request for this, instead try using alias inside

`PublicationOperationFields.graphql` and `PublicationStatsFields.graphql`

Reply:
So the issue is that aliases aren't fully supported by graphql-codegen. See [this issue](https://github.com/dotansimha/graphql-code-generator/issues/1787). The query type is properly generated, but not the return type. In our case, `PublicationStats` doesn't have the aliased field (even though it's properly returned by the query).

[graphql-typescript-response-factories](https://github.com/homebound-team/graphql-typescript-response-factories) adds support for aliases but only at the top level, not in fragments (see [this issue](https://github.com/homebound-team/graphql-typescript-factories/issues/54)).

Any ideas on the best way to go about this?

Reply:
Thoughts:

- An alias isn't actually needed in `PublicationOperationFields` since `actedOn` is not currently used -- I can move this directly from `UnknownOpenActionData` to `PublicationOperationFields`.
- The publication details page could afford to make the separate query for the count, and keep the "Tippers" page.
- We could do some server-side caching of the tip counts, as a last resort.

If we're unable to show the tip count in the feed, the other parts of this PR are probably still worthwhile. If we cannot find a way to make codegen work with aliases in fragments, or a suitable workaround, caching the tip counts might be a viable option, though not particularly desirable.